oleas/helpers.py

```diff
@@ -48,18 +48,21 @@ def setup_logger_output():
     return logger
 
 
-def get_board(serial: str, model: str='aodsoc_aods', baud=None):
+def get_board(serial: str, model: str='aodsoc_aods', baud=None, config: str=None):
     """Connect to a board
 
     Args:
         serial (str): the board serial number
         model (str): board model
         baud (int): baud rate, or None to use fastest available.
+        config (str): path to config file
 
     Returns:
         _type_: _description_
     """
     board = Board(model)
+    if config:
+        board.load_registers(config)
```

Comment:
```suggestion
    board = Board(model, registers=config)
```
registers is part of the board init and will load default on `None`